src/Core/src/Handlers/WebView/WebViewHandler.Windows.cs

```diff
@@ -366,6 +371,10 @@ void OnCoreWebView2Initialized(WebView2 sender, CoreWebView2InitializedEventArgs
 				if (Handler is WebViewHandler handler)
 				{
 					sender.UpdateUserAgent(handler.VirtualView);
+				    if (handler.PlatformView.Source is not null)
```

Comment:
Could simplify the code using directly `sender` instead `handler.PlatformView`.
For example:

`if(sender.Source is not null)`

Reply:
> Could simplify the code using directly `sender` instead `handler.PlatformView`. For example:
> 
> `if(sender.Source is not null)`

Hi @jsuarezruiz,
We have used the Microsoft Learn documentation URL: https://learn.microsoft.com/en-us/dotnet/.